test/multiple_config.js

```diff
@@ -0,0 +1,8 @@
+export default function () {
```

Comment:
__question:__ this file isn't needed anymore, right (:tada:)?

Reply:
Yes, I thought I deleted it!

Reply:
That's strange... it is deleted and I see a commit for it. Hmm https://github.com/elastic/kibana/pull/18568/commits/32037a780e9d06046966d38f6a0113bdcaaee5a6

Reply:
Ah, it is deleted from x-pack but not from kibana.